openshift/unicorn-base/openshift/build.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: v1
+kind: Template
+metadata:
+  creationTimestamp: null
+  name: uvicorn-gunicorn-fastapi
+labels:
+  app: ${NAME}${SUFFIX}
+  phase: build
+  app.kubernetes.io/name: uvicorn-gunicorn-fastapi
+  app.kubernetes.io/managed-by: template
+  app.kubernetes.io/version: "python3.8"
+parameters:
+  - name: NAME
+    value: uvicorn-gunicorn-fastapi
+  - name: SUFFIX
```

Comment:
This should be required I guess?

Reply:
Seems like doesn't have to be!

Reply:
Nope - I didn't even use it. The suffix makes sense when we're differentiating between PR and prod builds in the way we build images. We're not actually even using the suffix here. Right now we've got: uvicorn-gunicorn-fastapi   docker-registry.default.svc:5000/auzhsi-tools/uvicorn-gunicorn-fastapi   python3.8-latest,python3.8-TLSv1.0 (so calling it uvicorn-gunicorn-fastapi, and doing the differentiating with tags)